docs/features/nullable-reference-types.md

```diff
@@ -175,6 +175,11 @@ A `class?` constraint is allowed, which, like class, requires the type argument
 [Nullable strawman](https://github.com/dotnet/csharplang/issues/790)
 [4/25/18](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-04-25.md)
 
+An explicit `object` constraint is allowed, which requires the type to be non-nullable when it is a reference type.
```

Comment:
>`object` [](start = 12, length = 8)

Although I understand `object` is emitted just like `System.Object` constraint, or any other type constraint, it may be good to call out for our F#/interop friends.